hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcDuplexHandler.java

```diff
@@ -103,8 +103,8 @@ private void writeRequest(ChannelHandlerContext ctx, Call call, ChannelPromise p
         ctx.write(buf, withoutCellBlockPromise);
         ChannelPromise cellBlockPromise = ctx.newPromise();
         ctx.write(cellBlock, cellBlockPromise);
-        PromiseCombiner combiner = new PromiseCombiner();
-        combiner.addAll(withoutCellBlockPromise, cellBlockPromise);
+        PromiseCombiner combiner = new PromiseCombiner(ctx.executor());
```

Comment:
This a bug fix? Passing the executor?

Reply:
The default constructor has been deprecated.

https://github.com/netty/netty/blob/3e8e52725bbd9d999873880956bc8f241372daa8/common/src/main/java/io/netty/util/concurrent/PromiseCombiner.java#L73